Packing nuget failed

The `dotnet pack` command failed to produce a nuget package.

```
PS C:\Users\jason.jia\Source\Repos\lf-api-client-core-dotnet\src> dotnet pack -c release
Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
C:\Program Files\dotnet\sdk\6.0.201\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): error NU5026: The file 'C:\Users\jason.jia\Source\Repos\lf-api-client-core-dotnet\src\bin\release\net6.0\Laserfiche.Api.Client.Core.dll' to be packed was not found on disk. [C:\Users\jason.jia\Source\Repos\lf-api-client-core-dotnet\src\Laserfiche.Api.Client.Core.csproj]
```

--- a/src/Laserfiche.Api.Client.Core.csproj
+++ b/src/Laserfiche.Api.Client.Core.csproj
@@ -23,7 +23,7 @@
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <RepositoryUrl>https://github.com/Laserfiche/lf-api-client-core-dotnet</RepositoryUrl>
     <RepositoryType>GitHub</RepositoryType>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
     <AssemblyVersion>1.0.1.0</AssemblyVersion>
     <FileVersion>1.0.1.0</FileVersion>
   </PropertyGroup>

